### PR TITLE
Get college and group dropdown api's

### PR DESCRIPTION
**Pull request by Shreya-Kanani**

### CODE REVIEW

File: APTITUDE/APTITUDETEST.Application/Services/CollegeService.cs [hunk 1]
```diff
@@ -24,6 +24,11 @@ public async Task<JsonResult> GetColleges(CollegeQueryVM collegeQuery)
             return await _repository.GetColleges(collegeQuery);
         }
 
+        public async Task<JsonResult> GetCollegesForDropDown()
```
Comment (ShubhamPadiyaTatvasoft) [inline APTITUDE/APTITUDETEST.Application/Services/CollegeService.cs:27]: change name to GetActiveColleges

File: APTITUDE/APTITUDETEST.Application/Services/GroupService.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+        #region Constructor
+        public GroupService(IGroupRepository repository)
+        {
+            _repository = repository;
+        }
+        #endregion
+
+        #region Methods
+        public async Task<JsonResult> GetGroupsForDropDown()
```
Comment (ShubhamPadiyaTatvasoft) [inline APTITUDE/APTITUDETEST.Application/Services/GroupService.cs:20]: name change to same as colleges

File: APTITUDE/APTITUDETEST.Data/Data/CollegeRepository.cs [hunk 3]
```diff
@@ -68,6 +68,48 @@ public async Task<JsonResult> GetColleges(CollegeQueryVM collegeQuery)
 
         }
 
+        public async Task<JsonResult> GetCollegesForDropDown()
+        {
+
+            try
+            {
+                var collegeList = await Task.FromResult(_context.MasterCollege
+            .Where(x => x.IsDeleted == null || x.IsDeleted == false)
```
Comment (ShubhamPadiyaTatvasoft) [inline APTITUDE/APTITUDETEST.Data/Data/CollegeRepository.cs:77]: there is no filtered data for active or inactive colleges.

File: APTITUDE/APTITUDETEST.Data/Data/CollegeRepository.cs [hunk 4]
```diff
@@ -68,6 +68,48 @@ public async Task<JsonResult> GetColleges(CollegeQueryVM collegeQuery)
...
+                        Result = true,
+                        StatusCode = ResponseStatusCode.Success
+                    });
+                }
+                else
+                {
+                    return new JsonResult(new ApiResponse<string>
+                    {
+                        Data = "No College found",
+                        Message = ResponseMessages.Success,
```
Comment (ShubhamPadiyaTatvasoft) [inline APTITUDE/APTITUDETEST.Data/Data/CollegeRepository.cs:96]: use static messages for data.

File: APTITUDE/APTITUDETEST.Data/Data/GroupRepository.cs [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+            _context = context;
+        }
+        #region Methods
+        public async Task<JsonResult> GetGroupsForDropDown()
+        {
+            try
+            {
+                var collegeList = await Task.FromResult(_context.MasterGroup
+            .Where(x => x.IsDeleted == null || x.IsDeleted == false)
+            .Select(x => new { Id = x.Id, Name = x.Name })
```
Comment (ShubhamPadiyaTatvasoft) [inline APTITUDE/APTITUDETEST.Data/Data/GroupRepository.cs:30]: check active inactive same as colleges

**Review by mittalvyas1 — APPROVED** [review]

**Pull request merged**